packages/waku/src/lib/middleware/rsc.ts

```diff
@@ -121,7 +121,10 @@ export function rsc<Context>(options: {
   return async (req, res, next) => {
     const config = await configPromise;
     const basePrefix = config.basePath + config.rscPath + '/';
-    const pathStr = req.url || '';
+    let pathStr = req.url || '';
+    if (pathStr.startsWith('/@fs/')) {
+      pathStr = pathStr.slice('/@fs/'.length);
+    }
```

Comment:
This is DEV-only, so only do it if `command==="dev"`.
Though, I'm not 100% sure if it's the right way to trim `/@fs` here.